qdrant/v0.10.x/collections.md

```diff
@@ -52,6 +53,44 @@ In addition to the required options, you can also specify custom values for the
 - `shard_number` - which defines how many shards the collection should have. See [distributed deployment](../distributed_deployment#sharding) section for details.
 - `on_disk_payload` - defines where to store payload data. If `true` - payload will be stored on disk only. Might be useful for limiting the RAM usage in case of large payload.
 
+*Available since v0.10.0*
+
+There is support for multiple vectors per record. This feature allows for content multiple vector storages per collection. To separate different vectors in one record, each vector should have a unique name that should be defined while collection creation at once. Each named vector in this mode has its own distance and size:
```

Comment:
I find this paragraph a bit heavy, trying to make this read a bit smoother.

```suggestion
It is possible to have multiple vectors per record. This feature allows for multiple vector storages per collection. To separate different vectors in one record, each vector should have a unique name that should be defined when creating the collection. Each named vector in this mode has its own distance and size:
```

Reply:
Thanks a lot!